packages/react-native-sdk/src/contexts/CallCycleContext.tsx

```diff
@@ -12,9 +12,10 @@ import {
   useState,
 } from 'react';
 import { useCallCycleEffect } from '../hooks';
+import { useIsOnline } from '../hooks/useIsOnline';
 
 /**
```

Comment:
This context has been removed, this component can be mounted/used somewhere else.
Maybe a utility hook dedicated for connectivity or [useCallCycleEffect](https://github.com/GetStream/stream-video-js/blob/main/packages/react-native-sdk/src/hooks/useCallCycleEffect.tsx)